concourse/pipelines/partner-image-export.yaml

```diff
@@ -0,0 +1,1046 @@
+    vars:
+      wf: "partner/ubuntu_pro_1604_export.wf.json"
+      gcs_url: ((.:gcs-url))
+      build_date: ((.:build-date))
+# Ubuntu Pro 18.04
+- name: export-ubuntu-pro-1804
+  plan:
+  - get: compute-image-tools
+  - get: guest-test-infra
+  - task: generate-build-id
+    file: guest-test-infra/concourse/tasks/generate-build-id.yaml
+    vars:
+      prefix: "ubuntu-pro-1804"
+  - put: ubuntu-pro-1804-gcs
+    params:
+      file: build-id-dir/ubuntu-pro-1804*
+    get_params:
+      skip_download: "true"
+  - load_var: gcs-url
+    file: ubuntu-pro-1804/url
+  - task: generate-build-date
+    file: guest-test-infra/concourse/tasks/generate-version.yaml
+  - load_var: build-date
+    file: publish-version/version
+  - task: get-credential
+    file: guest-test-infra/concourse/tasks/get-credential.yaml
+  - task: daisy-export-ubuntu-pro-1804
+    file: guest-test-infra/concourse/tasks/daisy-export-images-partner.yaml
+    vars:
+      wf: "partner/ubuntu_pro_1804_export.wf.json"
+      gcs_url: ((.:gcs-url))
+      build_date: ((.:build-date))
+# Ubuntu Pro 20.04
+- name: export-ubuntu-pro-2004
+  plan:
+  - get: compute-image-tools
+  - get: guest-test-infra
+  - task: generate-build-id
+    file: guest-test-infra/concourse/tasks/generate-build-id.yaml
+    vars:
+      prefix: "ubuntu-pro-2004"
+  - put: ubuntu-pro-2004-gcs
+    params:
+      file: build-id-dir/ubuntu-pro-2004*
+    get_params:
+      skip_download: "true"
+  - load_var: gcs-url
+    file: ubuntu-pro-2004/url
+  - task: generate-build-date
+    file: guest-test-infra/concourse/tasks/generate-version.yaml
+  - load_var: build-date
+    file: publish-version/version
+  - task: get-credential
+    file: guest-test-infra/concourse/tasks/get-credential.yaml
+  - task: daisy-export-ubuntu-pro-2004
+    file: guest-test-infra/concourse/tasks/daisy-export-images-partner.yaml
+    vars:
+      wf: "partner/ubuntu_pro_2004_export.wf.json"
+      gcs_url: ((.:gcs-url))
+      build_date: ((.:build-date))
+
+
+# Images for OS Loging staging tests.
+# COS 81 LTS
+- name: publish-oslogin-cos-81-lts
+  plan:
+  - get: guest-test-infra
+  - get: compute-image-tools
+  - get: cos-81-lts-gcs
+    passed: [export-cos-81-lts]
+    trigger: true
+    params:
+      skip_download: "true"
+  - load_var: source-version
+    file: cos-81-lts-gcs/version
+  - task: get-credential
+    file: guest-test-infra/concourse/tasks/get-credential.yaml
+  - task: generate-version
+    file: guest-test-infra/concourse/tasks/generate-version.yaml
+  - load_var: publish-version
+    file: publish-version/version # produced from generate-version task
+  - task: publish-cos-81-lts
+    file: guest-test-infra/concourse/tasks/daisy-publish-images.yaml
+    vars:
+      source_gcs_path: "gs://gce-image-archive/partner/cos"
+      source_version: v((.:source-version))
+      publish_version: ((.:publish-version))
+      wf: "partner/cos_81_lts.publish.json"
```

Comment:
wf resolves to compute-image-tools/daisy_workflows/build-publish/partner/cos_81_lts.publish.json, but i don't see that in the repository or this PR. does this PR depend on another ?

Reply:
Yes, it depends on https://github.com/GoogleCloudPlatform/compute-image-tools/pull/1601